theme/errorList.less

```diff
@@ -7,6 +7,10 @@
     min-height: @errorListHeight;
     border-top: 2px @errorListBorderColor solid;
 
+    *:focus {
+        outline: none;
```

Comment:
could we see a gif of how it looks w/o this line? The focus is pretty subtle in non high contrast mode, and blended in with the zebra striping a bit to my eyes

Reply:
Here's what it looks without it The one thing was that the outline was always persistent
![tab_nav_outline](https://user-images.githubusercontent.com/46362603/83816321-fc737c00-a676-11ea-94ba-03de9323e395.gif)

Reply:
Hm, I suppose that's a bit aggressive when you're clicking on it (though I think I might like it for the errors themselves).

Maybe just a tiny bit darker for focus to differentiate it a bit more? Assuming you can do that without the contrast with the text becoming an issue